### PR TITLE
Minor UX fixes in assisted tagging and assisted mapping page

### PR DESCRIPTION
**Pull request by AdityaKhatri**
## Changes

* Add draftEntry to entry
* Use real assistedTaggingEnabled boolean

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: app/components/framework/CompactAttributeInput/GeoLocationWidgetInput/index.tsx [hunk 1]
```diff
@@ -35,6 +40,8 @@ export interface Props <N extends string>{
     onGeoAreaOptionsChange: React.Dispatch<React.SetStateAction<GeoArea[] | undefined | null>>;
 
     widgetHints?: string[];
+    suggestionModeEnabled?: boolean;
```
Comment (tnagorra) [inline app/components/framework/CompactAttributeInput/GeoLocationWidgetInput/index.tsx]:
```suggestion
    suggestionMode?: boolean;
```

File: app/views/ProjectEdit/Connector/schema.ts [hunk 2]
```diff
@@ -29,34 +33,65 @@ export const sourceSchema:SourceFormSchema = {
...
-            // FIXME: Define better schema
-            params: [],
         };
 
         switch (value?.source) {
-            case 'RELIEF_WEB':
+            // TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO:
+            // - review code changes
+            // - also call mom
+            // - or look at note
```
Comment (tnagorra) [inline app/views/ProjectEdit/Connector/schema.ts]: Let's not comment personal todo on deep source code.

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/views/ProjectEdit/Connector/schema.ts (hunk 2 above)
Comment (frozenhelium) [reply app/views/ProjectEdit/Connector/schema.ts]: Yes please, let's use separate app or a private repo for that

Comment (frozenhelium) [reply app/views/ProjectEdit/Connector/schema.ts]: Let's also do one round of quick self review before assigning it to somebody else

**Pull request merged**